justfile

```diff
@@ -0,0 +1,20 @@
+# Justfile
+
+default:
+    help
+
+# Help task to display available tasks
+help:
+    @echo "Available tasks:"
+    @echo "  test       - Run tests using pytest."
+    @echo "  precommit  - Run pre-commit checks."
+
+# Task to run tests using pytest
+test:
+    @echo "Running tests..."
```

Comment:
before running test, fastapi server should run in another shell.
workout this.
because right now some tests would fail with this logic